packages/wallet/frontend/src/styles/main.css

```diff
@@ -37,7 +37,7 @@
 }
 
 #__next {
-  @apply md:grid md:grid-cols-[max-content_1fr] relative;
+  @apply h-full;
```

Comment:
Let's revert these changes. We will update the styles accordingly while re-designing. Let's just update the images in this PR.